### ISSUE TITLE
Update the Admin portal within ZooKeepr to link to hidden reports

### ISSUE THREAD

**Issue by KathyReid** [zookeepr/zookeepr#364]
### User story

As an Administrator of ZooKeepr, I want to be able to see all the reports available to me as an Administrator so that I can effectively run a conference, and not have to bookmark secret hidden reports, thus saving me from become a mad alcoholic. 

### Acceptance

* The admin portal at $LINK is updated with links to the following reports
* $REPORT - $LINK

@jhesketh sorry can someone get me a list of links and reports and I can flesh out this issue a bit more, thanks

**Comment by jhesketh** [comment]
I'm not sure we have a list. The problem is that the /admin page might not be complete.

There are a few items here todo:

 * Get rid of the bootstrap admin theme as it was experimental and doesn't do anything useful (keep the checkin stuff, but separately).
 * Format the admin-index nicely
 * Audit any missing pages from the admin-index. I'm thinking controllers that have been added post LCA09 (when I made the admin page) such as funding and volunteers etc.

TL;DR: Neaten the admin page and add links to other pages if missing.
